src/v/cluster/config_manager.h

```diff
@@ -12,7 +12,9 @@
 #pragma once
 
 #include "cluster/commands.h"
+#include "cluster/members_table.h"
```

Comment:
Could just use `cluster/fwd.h` as the header only uses a pointer to members table.